SQL_Assignments/sql_4_a.txt

```diff
@@ -0,0 +1,49 @@
+-- Questions
+
+-- 1. List all patients and their appointments, including patients who have never had an appointment.
```

Comment:
Always append the TABLE schema. It would be helpful to get the complete context of the queries.

Reply:
ok will do.